README.md

```diff
@@ -37,13 +37,12 @@ git clone https://gitlab.pnnl.gov/exasgd/solvers/hykkt.git
 Make sure you have following build dependencies installed:
 * C++ compiler supporting C++11 standard or higher
 * CMake >= 3.19
-* CUDA >= 11.0
+* 12.0 > CUDA >= 11.0
 
 To build HyKKT library and drivers simply
 ```
-mkdir build
 cd build
-cmake ../hykkt
+cmake ..
```

Comment:
The current instructions don't work because hykkt is the parent directory (this implies it's a sibling directory of sorts)

Reply:
You still cd into build and then build there, so that's fine

Reply:
> The current instructions don't work because hykkt is the parent directory (this implies it's a sibling directory of sorts)

This is correct -- `build` in these instructions is a sibling directory to `hykkt`.

Reply:
Oh, this is confusing because there is a `build` directory within `hykkt` as well. Also, typically instructions for using and installation start from within the directory being cloned. Put differently, `cd hykkt` is implied. Although maybe I have experience only with incorrectly documented programs, but it's what I've grown to expect.